scripts/validate_test_schemas.py

```diff
@@ -0,0 +1,131 @@
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from structlog import get_logger
+
+logger = get_logger()
+
+error = False
+passed = 0
+failed = 0
+
+
+def validate_schema(schema_path):
+    try:
+        result = subprocess.run(
+            [
+                "curl",
+                "-s",
+                "-w",
+                "HTTPSTATUS:%{http_code}",
+                "-X",
+                "POST",
+                "-H",
+                "Content-Type: application/json",
+                "-d",
+                f"@{schema_path}",
+                "http://localhost:5001/validate",
+            ],
+            capture_output=True,
+            text=True,
+            check=True,
+        )
+        return schema_path, result.stdout
+    except subprocess.CalledProcessError as e:
+        logger.info(f"Error validating schema {schema_path}: {e}")
+        return schema_path, None
+
+
+def main():
+    # pylint: disable=global-statement, broad-exception-caught
```

Comment:
Think we can avoid using globals by setting up a class for these vars.  Also makes sense with the comment above if we're importing this as a class from the Validator repo 👍

Reply:
I moved them into the `main` now so we avoid globals, not sure if we want to move it into a class if we don't decide to move the script altogether.